test/other/test_standalone_syscalls.cpp

```diff
@@ -40,6 +40,6 @@ int main() {
   // This doesn't do anything because we have not handler registered, but it
```

Comment:
I just saw a typo here "not" -> "no".. only if you happen to update this PR again

Reply:
Fixed - might as well to trigger rerun.